api/api.go

```diff
@@ -136,6 +136,10 @@ func getContainersJSON(c *context, w http.ResponseWriter, r *http.Request) {
 		if !strings.Contains(tmp.Status, "Up") && !all {
 			continue
 		}
+		// Skip swarm containers unless -a was specified.
+		if string.Split(strings.TrimPrefix(tmp.Image, "library/"), ":")[0] == "swarm" && !all {
```

Comment:
I'm not sur, I believe the `library/` is stripped by docker already. Let's wait for the official image